docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -1,74 +1,88 @@
 ---
 id: sign-a-plugin
 title: Sign a plugin
-sidebar_position: 2
+sidebar_position: 4
+description: "How to sign a Grafana plugin."
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - sign plugin
+  - signing plugin
 ---
 
-<head>
-  <meta name="robots" content="noindex" />
-</head>
+# Sign a plugin
 
-Grafana requires all plugins to be signed so that we can verify their authenticity with signature verification.
+Grafana requires all plugins to be signed so that we can verify their authenticity with [signature verification](https://grafana.com/docs/grafana/latest/administration/plugin-management#plugin-signatures).
 
-All Grafana Labs-authored backend plugins, including Enterprise plugins, are signed. By default, Grafana **requires** all plugins to be signed in order for them to be loaded.
+All Grafana Labs-authored backend plugins, including Enterprise plugins, are signed. By [default](https://grafana.com/docs/grafana/latest/administration/plugin-management#allow-unsigned-plugins), Grafana **requires** all plugins to be signed in order for them to be loaded.
 
 Before you can sign your plugin, you need to decide whether you want to sign it as a _public_ or a _private_ plugin.
 
-To make your plugin publicly available outside of your organization, sign your plugin under a _community_ or _commercial_ signature level. Public plugins are available from the Grafana plugin catalog and can be installed by anyone.
+To make your plugin publicly available outside of your organization, sign your plugin under a _community_ or _commercial_ [signature level](#plugin-signature-levels). Public plugins are available from the [Grafana plugin catalog](https://grafana.com/plugins) and can be installed by anyone.
 
-If you intend to only use the plugin within your organization, sign it under a _private_ signature level.
+If you intend to only use the plugin within your organization, sign it under a _private_ [signature level](#plugin-signature-levels).
 
-## Generate an API key
+## Generate a token
```

Comment:
follow up - maybe we should look at the exact terminology used here, as you need to generate a cloud access policy and use a token for that policy.

Reply:
```suggestion
## Generate an Access Policy token
```

Reply:
Fixed.